Workspace/src/gimbal_control/Camera2.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+from sensor_msgs.msg import Joy
+from can_msgs.msg import Frame
+import rospy
+import struct
+
+
+
+class GimbalController:
+
+	def __init__(self):
+
+		self.init_camera=rospy.init_node("Gimbal", anonymous=True)
+		self.sub_camera=rospy.Subscriber("/joy",Joy,self.callback) #receives Xbox commands by subscribing to Joy topic
+		self.pub_camera= rospy.Publisher("/gimbal_camera", Frame, queue_size=20) #xbox publishes to the camera
+		self.pan=0
+		self.tilt=0
+		self.ID = 600 # CAN id
+
+
+	def callback(self, joy):
+
+		if joy.axes[5]==1:
+			self.tilt+=10
```

Comment:
can you make this number 10 tunable using a config param? ros has something called rosparams which you can use for this purpose.